src/Chapter09/Listing09.22.OverridingGetHashCodeUsingCombine.cs

```diff
@@ -0,0 +1,53 @@
+using System.Runtime.CompilerServices;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_22;
+
+public class Angle
+{
+    public int Degrees { get; init; }
+    public int Minutes { get; init; }
+    public int Seconds { get; init; }
+    public string? Name { get; init; }
+    public Angle(int Degrees, int Minutes, int Seconds, string? Name)
+    {
+        this.Degrees = Degrees;
+        this.Minutes = Minutes;
+        this.Seconds = Seconds;
+        this.Name = Name;
+    }
+
+    public Angle(int Degrees, int Minutes, int Seconds)
+    {
+        this.Degrees = Degrees;
+        this.Minutes = Minutes;
+        this.Seconds = Seconds;
+        this.Name = null;
+    }
+
+    #region INCLUDE
+    public override int GetHashCode() =>
+    HashCode.Combine(Degrees,
+        Minutes, Seconds);
```

Comment:
nit: Spacing. I realize this is how it is in the book, but I think a little space makes it a bit easier to read.
```suggestion
    public override int GetHashCode() =>
        HashCode.Combine(Degrees, Minutes, Seconds);
```